src/renault_api/cli/renault_vehicle.py

```diff
@@ -315,6 +316,27 @@ async def update_lock_status(
             update_status_table(status_table, key, value, unit)
 
 
+async def update_res_state(
+    vehicle: RenaultVehicle, status_table: Dict[str, Any]
+) -> None:
+    """Update status table from get_vehicle_res_state."""
+    try:
+        if not await vehicle.supports_endpoint("res-state"):  # pragma: no cover
+            return
+        response = await vehicle.get_res_state()
+    except QuotaLimitException as exc:  # pragma: no cover
+        raise click.ClickException(repr(exc)) from exc
+    except KamereonResponseException as exc:  # pragma: no cover
+        click.echo(f"res state: {exc.error_details}", err=True)
+    else:
+        items = [
+            ("Engine state", response.details, None),            
+        ]
```

Comment:
There are some trailing spaces that made their way in...
```suggestion
        items = [
            ("Engine state", response.details, None),
        ]
```

Reply:
yeah, fixed in the next commit